db/seeds.rb

```diff
@@ -1,8 +1,12 @@
 # frozen_string_literal: true
 
-Dir[Rails.root.join("db/seeds/initial_seed.rb")].each { |seed| load seed }
+Dir[Rails.root.join("db/seeds/initial_seed.rb")].each { |seed| load seed } unless Rails.env.sandbox?
```

Comment:
I think we should have the same seeds in all non-prod environments. I'm not sure if there's a reason that the other seed data would cause problems in the sandbox. The extra mental/code complexity worries me.

Reply:
done